src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -75,13 +77,25 @@ CAC.Pages.Home = (function ($, ModeOptions,  MapControl, TripOptions, SearchPara
         modeOptionsControl = new ModeOptions();
         modeOptionsControl.setMode(UserPreferences.getPreference('mode'));
 
+        directionsFormControl = new CAC.Control.DirectionsFormControl({});
+
         directionsControl = new CAC.Control.Directions({
             mapControl: mapControl,
+            directionsFormControl: directionsFormControl,
+            tabControl: tabControl,
+            urlRouter: urlRouter
+        });
+
+        exploreControl = new CAC.Control.Explore({
+            mapControl: mapControl,
+            modeOptionsControl: modeOptionsControl,
```

Comment:
I think we should be able to leave the `modeOptionsControl` out of the `exploreControl`, as it has been removed from the `directionsControl`, and instead modify the mode toggle listener on this home page module to check the tab and call appropriately into either explore or directions control.